allennlp/modules/token_embedders/token_characters_encoder.py

```diff
@@ -0,0 +1,40 @@
+import torch
+
+from allennlp.common import Params
+from allennlp.data.vocabulary import Vocabulary
+from allennlp.experiments import Registry
+from allennlp.modules.token_embedders import Embedding
+from allennlp.modules import Seq2VecEncoder, TimeDistributed, TokenEmbedder
+
+
+@Registry.register_token_embedder("character_encoding")
+class TokenCharactersEncoder(TokenEmbedder):
```

Comment:
We're blurring the lines a bit here between the boundary of `Embedders` and `Encoders`, but equally, I can't think of a better way ...